style.css

```diff
@@ -0,0 +1,448 @@
+  opacity: 0.3;
+  background-image: url('./imges/header-bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.cover h1 {
+  position: relative;
+  font-family: var(--firstFont);
+  color: var(--orangeColor);
+  font-size: 1em;
+}
+
+.cover h1 span {
+  font-size: 1.3em;
+}
+
+.cover p {
+  position: relative;
+  border: 1px solid rgb(255, 255, 255);
+  margin-right: 1em;
+  padding: 0.3em;
+  font-family: var(--secondFont);
+  font-size: 1em;
+  color: var(--darkColor);
+  background-color: rgba(255, 255, 255, 0.479);
+  border-radius: 3px;
+}
+
+.cover h3 {
+  position: relative;
+  font-family: var(--firstFont);
+  font-size: 2em;
+  color: var(--lightColor);
+  background-color: rgba(54, 54, 54, 0.5);
+  width: fit-content;
+  margin: 0.4em;
+  border-radius: 3px;
+  padding: 0.4em;
+}
+
+.cover h6 {
+  position: relative;
+  color: var(--lightColor);
+  font-family: var(--secondFont);
+  font-size: 1em;
+  background-color: rgba(54, 54, 54, 0.5);
+  width: fit-content;
+  margin: 0.4em;
+  border-radius: 3px;
+  padding: 0.4em;
+}
+
+/* speakers */
+
+.speakers img {
+  width: 100%;
+  padding: 2em;
+}
+
+.speaker-header {
+  color: var(--darkColor);
+  font-family: var(--firstFont);
+  font-size: 3em;
+  text-align: center;
+  background-color: rgb(85, 80, 80);
+}
+
+.speakers h1 {
+  font-family: var(--firstFont);
+  color: var(--orangeColor);
+  text-align: center;
+  font-size: 1.6em;
+}
+
+.line {
+  color: var(--orangeColor);
+  height: 20px;
+  stroke-width: 2;
+}
+
+.speakers p {
+  font-family: var(--secondFont);
+  color: var(--lightColor);
+  text-align: center;
+  font-size: 1.2em;
+}
+
+/* programs styling */
+
+.programs {
+  background-color: var(--darkColor);
+  padding-bottom: 2em;
+}
+
+.programs h2 {
+  color: #fff;
+  font-family: var(--firstFont);
+  text-align: center;
+  font-size: 1.5em;
+  padding: 1em;
+  font-weight: 600 !important;
+}
+
+.programs ul {
+  display: flex;
+  flex-direction: column;
+}
+
+.programs ul li {
+  display: flex;
+  align-items: center;
+  margin: 1em;
+  background: rgba(255, 255, 255, 0.068);
+  padding: 1em;
+  flex: 1;
+}
+
+.speakers {
+  padding: 2em;
+  background-color: rgb(85, 80, 80);
+}
+
+.programs ul li img {
+  flex: 1;
+}
+
+.programs ul li h3 {
+  color: var(--orangeColor);
+  font-family: var(--secondFont);
+  font-size: 1.6em;
+  margin: 0.5em;
+  font-weight: 600 !important;
+  flex: 2;
+}
+
+.programs ul li p {
+  color: #fff;
+  margin: 0.2em;
+  flex: 4;
+  font-size: 0.9em;
+  text-align: center;
+}
+
+.programs h4 {
+  background-color: var(--orangeColor);
+  font-family: var(--firstFont);
+  font-size: 0.9em;
+  letter-spacing: 2px;
+  font-weight: 400 !important;
+  color: #fff;
+  padding: 1.4em;
+  margin: 0 auto;
+  text-align: center;
+  text-decoration: none;
+  width: max-content;
+  cursor: pointer;
+}
+
+.programs h4:hover {
+  background-color: #b1392c;
+}
+
+/* disktop */
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Please always keep this in mind:
The CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around.

- [ ] **OPTIONAL**
It's always better to have at least two media queries for mid-sized and large-sized screens.

Reply:
hello Mahmoud 
my site is a mobile-first I don't understand what should I do about this

Reply:
yes, I'm sorry about that. it's just the optional comment you should kindly consider.